db/schema.rb

```diff
@@ -195,6 +195,12 @@
     t.integer  "formal_education_id"
     t.boolean  "email_notification",     default: true
     t.boolean  "adhoc",                  default: true
+    t.string   "NRIC_No"
```

Comment:
we usually don't use capital letters on column names. In this case, it would be `nric_no`